CIPs/CIP-draft_tanzanite.md

```diff
@@ -0,0 +1,35 @@
+---
+cip: <to be assigned>
+title: Meta: Tanzanite.
+author: Peilun (@peilun-conflux)
+discussions-to: https://forum.conflux.fun/t/topic/4249/68
+status: Draft
+type: Spec Breaking
+created: 2020-12-08
+---
+
+<!--You can leave these HTML comments in your merged CIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new CIPs. Note that a CIP number will be assigned by an editor. When opening a pull request to submit your CIP, please use an abbreviated title in the filename, `CIP-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Abstract
+<!--A short (~200 word) description of the technical issue being addressed.-->
+
+This specifies the changes included in the hard fork named Tanzanite.
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Conflux platforms ([conflux-rust](https://github.com/Conflux-Chain/conflux-rust)).-->
+
+* Codename: Tanzanite
+* Assigned version (first element in `custom`): `[1]`
+* Activation: 
+    * Block height >= 3,615,000 on Mainnet
+    * Block height >= 4,990,000 on Testnet
+    * Block height >= 0 on future testnets
+* Included CIPs:
+    * CIP-38 (Reduce block base reward to 2 CFX)
+    * CIP-39 (Use the custom field for incompatible upgrade)
```

Comment:
Is "bn128" going to be included?

Reply:
It will, but @ChenxingLi mentions we have claimed in the spec that we do support it like EVM... So I'm not sure if there will be a CIP describing the change. 
If that CIP is proposed, we should add it here.

Reply:
As I recall, Tethys already has at least one transaction that triggers it and fails. If someone builds another client that enables it from the genesis, the state roots will diverge.

Reply:
Right, this is a spec-breaking change, so we set the parameter to activate "bn128" in the future after Tanzanite is enabled (at block number BN128_ENABLE_NUMBER=7,600,000, which is expected to be several hours after epoch number 3,615,000). I do not find a proper way to include this parameter here if there is no CIP describing the change... Though it should have a CIP to make possible other implementations compatible.

Reply:
I suggest you add it with a placeholder CIP number, then it can be added later. I don't think the fact that the two changes do not happen simultaneously is an issue.